akka-stream/src/main/mima-filters/2.5.11.backwards.excludes

```diff
@@ -0,0 +1 @@
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.collectType")
```

Comment:
that file name is off by 2, should be 2.5.9
https://github.com/akka/akka/blob/master/CONTRIBUTING.md#binary-compatibility